### PR TITLE
Session Storage + Timeout Validation

### PR DESCRIPTION
**Pull request by henryrossiter**
- Switch from local storage to session storage
    - Saves user data and keeps user logged in for the lifetime of the browser tab
- Add simple timeout validation to event
    - Prevents users from setting zero-valued timeouts

### CODE REVIEW

**Comment by henryrossiter** [comment]
Fixes #157 
Fixes #155 
Fixes #151

File: src/State/state.js [hunk 1]
```diff
@@ -116,7 +116,7 @@ export const storeConfig = {
   state,
   mutations,
   plugins: [
-    createPersistedState(),
+    createPersistedState({ storage: window.sessionStorage }),
```
Comment (ProNinoGit120) [inline src/State/state.js:119]: Did you switch from localStorage to sessionStorage to be persisted only until the window or tab is closed?
Or for something else?

Comment (henryrossiter) [reply src/State/state.js:119]: Yes, like you said, with sessionStorage the data is only persisted only until the window or tab is closed.

The reason I did this was to solve [issue 155](https://github.com/ChildMindInstitute/mindlogger-admin/issues/155).

Comment (ProNinoGit120) [reply src/State/state.js:119]: Got it.

**Pull request merged**